src/template/partials/footer.html

```diff
@@ -1,3 +1,95 @@
 <footer>
+  <div class="container">
+    <div
+      class="contact-container d-flex flex-column align-items-center justify-content-center"
+    >
```

Comment:
You should add padding left and right as well to manage the div in responsive

Reply:
done.